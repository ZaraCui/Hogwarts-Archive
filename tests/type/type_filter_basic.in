ADD COLLECTION tests/data/books_small.csv
TYPE Charm
EXIT

