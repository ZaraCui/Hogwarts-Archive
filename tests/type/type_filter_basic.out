user: 12 spellbooks successfully added.

user: Advanced Charms (Filius Flitwick)
Advanced Charms (Filius Flitwick)
Levitation Charms (Helga Hufflepuff)
Levitation Charms (Helga Hufflepuff)

user: Ending Archive process.
